inc/functions/pluggable.php

```diff
@@ -0,0 +1,168 @@
+<?php
+/**
+ * Notification pluggable functions overrides
+ *
+ * @package notificaiton
+ */
+
+if ( ! function_exists( 'wp_notify_moderator' ) && ! notification_get_setting( 'integration/emails/notify_moderator' ) ) {
+	/**
+	 * Override wp_notify_moderator
+	 *
+	 * @return void
+	 */
+	function wp_notify_moderator() {}
+}
+
+if ( ! function_exists( 'wp_notify_postauthor' ) && ! notification_get_setting( 'integration/emails/notify_post_author' ) ) {
+	/**
+	 * Override wp_notify_postauthor
+	 *
+	 * @return void
+	 */
+	function wp_notify_postauthor() {}
+}
+
+if ( ! function_exists( 'wp_password_change_notification' ) && ! notification_get_setting( 'integration/emails/password_change' ) ) {
+	/**
+	 * Override wp_password_change_notification
+	 *
+	 * @return void
+	 */
+	function wp_password_change_notification() {}
+}
+
+if ( ! function_exists( 'wp_new_user_notification' ) ) {
+	/**
+	 * Override wp_new_user_notification
+	 *
+	 * @param int    $user_id User ID.
+	 * @param bool   $deprecated Deprecated.
+	 * @param string $notify Notify.
+	 * @return void
+	 */
+	function wp_new_user_notification( $user_id, $deprecated = null, $notify = '' ) {
+		if ( notification_get_setting( 'integration/emails/new_user_to_admin' ) ) {
+			notification_new_user_notification_to_admin( $user_id, $notify );
+		}
+
+		if ( notification_get_setting( 'integration/emails/new_user_to_user' ) ) {
+			notification_new_user_notification_to_user( $user_id, $deprecated, $notify );
+		}
+	}
+}
+
+if ( ( ! notification_get_setting( 'integration/emails/password_forgotten_to_admin' ) || ! notification_get_setting( 'integration/emails/password_forgotten_to_user' ) ) && ! function_exists( 'dont_send_password_forgotten_email' ) ) :
+	/**
+	 * Override wp_new_user_notification
+	 *
+	 * @param bool $send Whether to send.
+	 * @param int  $user_id User ID.
+	 * @return bool $send
+	 */
+	function dont_send_password_forgotten_email( $send = true, $user_id = 0 ) {
+
+		$is_administrator = notification_user_is_administrator( $user_id );
+
+		if ( $is_administrator && ! notification_get_setting( 'integration/emails/password_forgotten_to_admin' ) ) {
+			return false;
+		}
+		if ( ! $is_administrator && ! notification_get_setting( 'integration/emails/password_forgotten_to_user' ) ) {
+			return false;
+		}
+
+		return $send;
+
+	}
+
+endif;
+
+add_filter( ' allow_password_reset', 'dont_send_password_forgotten_email' );
```

Comment:
Is this filter really working? You have space in the hook name. Also, please prefix the function with `notification_` or better use anonymous function